std_msgs/UInt8 command
std_msgs/Float32 speed_val